FindTown/FindTown/Presentation/HomeScene/Home/HomeViewController.swift

```diff
@@ -6,102 +6,273 @@
+            townTableView.leadingAnchor.constraint(equalTo: backView.leadingAnchor, constant: 16),
+            townTableView.trailingAnchor.constraint(equalTo: backView.trailingAnchor, constant: -16),
+            townTableView.bottomAnchor.constraint(equalTo: backView.bottomAnchor)
+        ])
     }
     
-    @objc func action() {
-        viewModel?.push()
+    override func setupView() {
+        filterResetButton.setImage(UIImage(named: "reset"), for: .normal)
+        filterResetButton.setTitle("초기화", for: .normal)
+        filterResetButton.configuration?.contentInsets.leading = 12
+        filterResetButton.configuration?.contentInsets.trailing = 12
+        filterResetButton.setTitleColor(FindTownColor.black.color, for: .normal)
+        filterResetButton.isHidden = true
+        
+        filterButton.setImage(UIImage(named: "filter"), for: .normal)
+        filterButton.changesSelectionAsPrimaryAction = false
+        filterButton.setTitle("필터", for: .normal)
+        filterButton.configuration?.contentInsets.leading = 12
+        filterButton.configuration?.contentInsets.trailing = 12
+        filterButton.setTitleColor(FindTownColor.black.color, for: .normal)
+        
+        backView.backgroundColor = FindTownColor.back2.color
+        
+        townTableView.rowHeight = 150
+        townTableView.estimatedRowHeight = 150
     }
     
-    @objc func action4() {
-        viewModel?.pop()
-    }
-
-    @objc func action2() {
-        viewModel?.present()
+    override func bindViewModel() {
+        
+        filterCollectionView.rx.setDelegate(self)
+            .disposed(by: disposeBag)
+        
+        // Input
+        
+        filterButton.rx.tap
+            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
+            .bind { [weak self] in
+                self?.viewModel?.input.filterButtonTrigger.onNext(())
+            }
+            .disposed(by: disposeBag)
+        
+        filterResetButton.rx.tap
+            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
+            .bind { [weak self] in
+                self?.FilterResetButtonHidden(true)
+                self?.viewModel?.input.resetButtonTrigger.onNext(())
+            }
+            .disposed(by: disposeBag)
+        
+        // Output
+        
+        viewModel?.output.searchFilterDataSource
+            .observe(on: MainScheduler.instance)
+            .bind(to: filterCollectionView.rx.items) { collectionView, row, item in
+                let cell = collectionView.dequeueReusableCell(
+                    withReuseIdentifier: FillterCollectionViewCell.reuseIdentifier,
+                    for: IndexPath(row: row, section: 0)
+                ) as! FillterCollectionViewCell
+                
+                cell.setupCell(item, row)
+                
+                return cell
+            }
+            .disposed(by: disposeBag)
```

Comment:
```suggestion
       viewModel?.output.searchFilterDataSource
            .observe(on: MainScheduler.instance)
            .bind(to: filterCollectionView.rx.items(cellIdentifier: FillterCollectionViewCell.reuseIdentifier,
                                                    cellType: FillterCollectionViewCell.self)) { index, item, cell in
                cell.setupCell(item, index)
            }
            .disposed(by: disposeBag)
```
이렇게 작성하면 dequeue를 작성하지 않아도 돼요..!!

Reply:
박수 세번.. 👍